wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/MessageType.kt

```diff
@@ -50,9 +50,6 @@ data class MessageType(
 ) : Type() {
   private var deprecated: Any? = null
 
-  val isExtensible: Boolean
```

Comment:
Fwiw longer term, Kotlin could do the same extension stuff as swift is planning.